src/components/resource-picker/wikimedia-commons-search.yml

```diff
@@ -0,0 +1,24 @@
+searchFileTypeLabel:
+  en: Search for
+  de: Suche nach
+searchFileType_image:
+  en: Image
+  de: Bild
+searchFileType_video:
+  en: Video
+  de: Video
+searchFileType_audio:
+  en: Audio
+  de: Audio
+searchFileType_pdf:
+  en: PDF
+  de: PDF
+initialFilePreviewHeader:
+  en: Currently selected file
+  de: Momentan ausgewählte Datei
+initialFilePreviewFooter:
+  en: In order to select a different file, start a new search using the search field above
+  de: Um eine andere Datei auszuwählen, starte eine neue Suche im Suchfeld oben
+loadMoreSearchResults:
+  en: load more
+  de: mehr laden
```

Comment:
We've got the translation
```
more:
  en: more ...
  de: mehr ...
```
as well, maybe we could streamline? "load more" is clearer, but I like the shorter version actually. And I think people know through common UX patterns out there that "more ..." is for loading more 🤔 Or I presume they know

Reply:
I streamlined it (and another place it was used, too).